frontend/packages/component-library/src/cms/heroBanner/HeroBanner.tsx

```diff
@@ -128,7 +128,7 @@ const HeroBanner: React.FC<HeroBannerProps> = ({
         {partner && (
           <span className={moduleStyles.heroBannerPartnerContainer}>
             {partner.title}
-            <Image {...partner.logo} />
+            <Image {...partner.logo} hasRoundedCorners={false} />
```

Comment:
@levadadenys heads up I updated the HeroBanner and SkinnyBanner "in partnership with" images too!